src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -170,9 +175,37 @@
                     height: calc(100% + #{$clr-baselineRem_0_5 - 1});
                     width: 1px;
                     top: -$clr-baselineRem_0_25 + 1;
-                    left: $clr-baselineRem_0_5;
+                    left: 0;
                     background-color: $clr-lighter-midtone-gray;
                 }
+
+                .separator-dragger {
```

Comment:
@hippee-lee oh that's much better. I will use that instead of the current class name. Thanks!